OpenRA.Mods.TS/Activities/VoxelHarvesterDockSequence.cs

```diff
@@ -17,24 +18,52 @@ namespace OpenRA.Mods.TS.Activities
 	public class VoxelHarvesterDockSequence : HarvesterDockSequence
 	{
 		readonly WithVoxelUnloadBody body;
+		readonly WithHarvesterUnloadOverlay spriteOverlay;
```

Comment:
Please call this `WithDockingOverlayInfo` to match the existing `WithDockingAnimationInfo`.